pkg/client/client.go

```diff
@@ -412,4 +412,35 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, allAddresses,
 	} else {
 		fmt.Fprintf(w, "Proxy Status:\tNo managed proxy redirect\n")
 	}
+
+	if sr.Hubble != nil {
+		var fields []string
+
+		state := sr.Hubble.State
+		if sr.Hubble.Msg != "" {
+			state = fmt.Sprintf("%s %s", state, sr.Hubble.Msg)
+		}
+		fields = append(fields, state)
+
+		if o := sr.Hubble.Observer; o != nil {
+			var observer []string
+
+			if o.MaxFlows > 0 {
+				observer = append(observer, fmt.Sprintf("Current/Max Flows: %d/%d (%.2f%%)",
+					o.CurrentFlows, o.MaxFlows, (float64(o.CurrentFlows)/float64(o.MaxFlows))*100))
+			}
+			if o.Uptime > 0 {
+				observer = append(observer, fmt.Sprintf("Flows/s: %.2f",
+					float64(o.SeenFlows)/time.Duration(o.Uptime).Seconds()))
+			}
+
+			fields = append(fields, strings.Join(observer, ", "))
+		}
+
+		if sr.Hubble.Metrics != nil {
+			fields = append(fields, fmt.Sprintf("Metrics: %s", sr.Hubble.Metrics.State))
+		}
+
+		fmt.Fprintf(w, "Hubble:\t%s\n", strings.Join(fields, "\t"))
+	}
```

Comment:
Should we have a branch that prints "Hubble: Disabled"?

Reply:
So if the local cilium-agent has this PR applied, it should print "Hubble: Disabled", since the server always populates `sr.Hubble`. The only case where it would not print a `Hubble: Disabled` line is when you use this client with and older version of cilium-agent which does have the `HubbleStatus` in `/healthz`.

Considering all many of the other branches in this file also don't print anything if their corresponding section is missing, I think it's better to keep it as.